server/src/test/java/io/javaclasses/filehub/api/FolderTest.java

```
@@ -31,8 +30,14 @@ void testEqualsAndHashCode() {
 
     @DisplayName("not pass null parameters.")
     @Test
-    void testPassingNullParameters() {
+    void testPassingNullParameters() throws NoSuchMethodException {
```

- [ ] I don't think that test should throw any exceptions.

---

class.getConstructor() throws NoSuchMethodException that is not relevant to catch here. The test case is to check whether the constructor of Folder rejects null or not. The test case is not to check the existence of the constructor.

---

The test should not throw any exceptions. You can catch it in the method body and just _fail_ the test.